### ISSUE TITLE
Research web accessibility (colour contrasts, typeface readability etc) and incorporate in redesign

### ISSUE THREAD

**Issue by cazcam34** [talentbeyondboundaries/tbbtalentv2#2126]
Some useful links:
https://webaim.org/techniques/css/
https://webaim.org/articles/contrast/
https://webaim.org/techniques/fonts/
https://coolors.co/contrast-checker/112a46-acc8e5
https://wave.webaim.org/ - a web browser tool to monitor accessibility

Last year we had a detailed accessibility review completed (see [here](https://docs.google.com/document/d/10QsEdLubSP1j3zZynTNLEyc2wPNTMVm7/edit?tab=t.0#heading=h.gjdgxs)), and implemented some recommendations. I think with the upcoming redesign we have the chance to review the UI accessibility as this has been brought up by both [candidates](https://refugeejobsmarket.slack.com/archives/C42S4AJ73/p1719487883561959) and staff (see Slack from Nourhan: https://refugeejobsmarket.slack.com/archives/C42S4AJ73/p1710763298751649).
 I think Nourhan is quite passionate about this so she could be a great person to involve. She gave some great suggestions (see slack [here](https://refugeejobsmarket.slack.com/archives/C42S4AJ73/p1710880431060939?thread_ts=1710776304.407709&cid=C42S4AJ73) or read below):
- Make sure the users can resize the text and zoom in on the content without losing functionality or readability. Create designs that are responsive and flexible, allowing for different text sizes and zoom levels.
- Offer audio descriptions or alternative text for images, graphs, and other visual content so that those who have low vision can access the content.
- Give users an option of a high-contrast mode to read text on low-contrast backgrounds. Provide users with an option of switching between the standard and high-contrast modes based on their preferences.
- Verify that all interactive elements including menus, sliders, and carousels can be operated and activated not only using a mouse but also the keyboard. Also, introduce logical tabbing and keyboard shortcuts for frequent actions.
- Enable users to personalize the user interface as per their preference and requirements. Allow users to change colors, fonts, contrast levels and layout settings to enhance accessibility and usability.
- Incorporate speech input and dictation ability so that the users can put text and control the interface with their voice if they're not that comfortable to type.

Also Naji has previously brought up accessibility (see [here](https://refugeejobsmarket.slack.com/archives/C42S4AJ73/p1686278441317699)) so I think he'd be another good person we could engage when making more accessible.

I think the main need for accessibility would be the candidate portal, but it'll also be good practise to incorporate into the admin portal. Especially as we are in a redesign stage it should be fairly easy to factor in accessible UI such as colours/fonts/headers/sizing.

**Comment by cazcam34** [comment]
This was incorporated into Hiba's design work, using contrast checkers to make sure there was enough contrast between colours. More to do for accessibility in phase 2.
